hoot-core/src/main/cpp/hoot/core/io/OsmApiChangesetElement.h

```diff
@@ -50,22 +50,25 @@ class ElementIdToIdMap;
 
 /** Object that matches an XML tag to its XML attributes */
 typedef QPair<QString, QXmlStreamAttributes> XmlObject;
+typedef std::vector<std::pair<QString, QString>> ElementAttributes;
+typedef std::pair<QString, QString> ElementTag;
+typedef std::vector<ElementTag> ElementTags;
 
 /** Changeset element abstraction for simplified nodes, ways, and relations */
-class XmlElement
+class ChangesetElement
```

Comment:
Since the elements no longer keep XML objects I changed the names to `Changeset*`.  That is the majority of the changes that clutter this PR.